pkgs/sops-install-secrets/main.go

```diff
@@ -20,36 +21,43 @@ import (
 	"github.com/Mic92/sops-nix/pkgs/sops-install-secrets/sshkeys"
 	agessh "github.com/Mic92/ssh-to-age"
 
+	"github.com/facebookgo/symwalk"
```

Comment:
The repo is already archived. I would just copy this one function rather than depending on external code: https://github.com/facebookarchive/symwalk/blob/42004b9f322246749dd73ad71008b1f3160c0052/walk.go#L19